src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoFeetPlace.java

```diff
@@ -0,0 +1,255 @@
+package me.zeroeightsix.kami.module.modules.combat;
+
+import me.zeroeightsix.kami.command.Command;
+import me.zeroeightsix.kami.module.Module;
+import me.zeroeightsix.kami.module.ModuleManager;
+import me.zeroeightsix.kami.setting.Setting;
+import me.zeroeightsix.kami.setting.Settings;
+import me.zeroeightsix.kami.util.BlockInteractionHelper;
+import me.zeroeightsix.kami.util.Wrapper;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockObsidian;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.item.EntityItem;
+import net.minecraft.entity.item.EntityXPOrb;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.item.ItemStack;
+import net.minecraft.network.play.client.CPacketEntityAction;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3d;
+
+import static me.zeroeightsix.kami.util.BlockInteractionHelper.*;
+
+/**
+ * Created 13 August 2019 by hub
+ * Updated 21 November 2019 by hub
+ */
+@Module.Info(name = "AutoFeetPlace", category = Module.Category.COMBAT)
+public class AutoFeetPlace extends Module {
+
+    private final Vec3d[] surroundTargets = {
+            new Vec3d(1, 0, 0),
+            new Vec3d(0, 0, 1),
+            new Vec3d(-1, 0, 0),
+            new Vec3d(0, 0, -1),
+            new Vec3d(1, -1, 0),
+            new Vec3d(0, -1, 1),
+            new Vec3d(-1, -1, 0),
+            new Vec3d(0, -1, -1),
+            new Vec3d(0, -1, 0)
+    };
+
+    private Setting<Boolean> triggerable = register(Settings.b("Triggerable", true));
+    private Setting<Integer> triggerableTimeoutTicks = register(Settings.i("Triggerable Timeout (Ticks)", 20));
+    private Setting<Integer> blockPerTick = register(Settings.i("Blocks per Tick", 4));
+    private Setting<Boolean> rotate = register(Settings.b("Rotate", true));
+    private Setting<Boolean> announceUsage = register(Settings.b("Announce Usage", false));
+    private Setting<Boolean> debugMessages = register(Settings.b("Debug Messages", false));
```

Comment:
Generally I don't really like 'announcer' settings like these. Debug messages are only the timeout message, which is accompanied by a 'Disabled!' message anyways. And the enabled/disabled setting is doing what the active modules GUI widget is supposed to be doing: telling you whether or not a setting is enabled.
Are these settings necessary?

Reply:
I gonna remove them.

Reply:
Removed [here](4684ed8d82b1b8ff14b637baa80abb4a7007a148).